### DISCUSSION TITLE
Using tags as span identifiers, in the Tracing UI

### DISCUSSION THREAD

**Discussion by wengelbrecht-grafana** [grafana/grafana#43707] [Observability]
Hello

I'm working on a project where we would like to use tags in a span, as the span identifier (when using the Tempo as the backend).
For example, right now the span will identify itself as "HTTP GET", but if there are hundreds of spans it is quite hard to find the right one. It would be preferable to identify spans by a tag (eg, http.url) rather. Visually when looking through all the spans one could find it much easier to to use/find the information you're looking for. 

For discussion purposes:

How would the UI handle this ? Eg, if the tag has a very long value like a url
Should it be configurable ?
Maybe a different way to filter ?

**Comment by bboreham** [comment]
Prior art, of a sort: https://github.com/jaegertracing/jaeger-ui/pull/703
(the UX to configure tags is terrible, but it may spark some discussion)

**Comment by joey-grafana** [comment]
Hello, this work was completed in https://github.com/grafana/grafana/issues/50931 :)
